install/samples/lazyload/easy_install_lazyload.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```

Comment:
tag_or_com=${TAG_OR_COMMIT}
if [[ -z "${tag_or_com}" ]]; then
  echo "should pass tag or commit via env TAG_OR_COMMIT" >&2
  exit 1
fi

Reply:
Done. Use latest_tag as default, user can add paramater to override, and use customize tag or commit version.